bedrock/base/templates/includes/canonical-url.html

```diff
@@ -10,7 +10,7 @@
         {% if code == 'en-US' -%}
         <link rel="alternate" hreflang="en" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English">
         <link rel="alternate" hreflang="en-CA" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English (Canada)">
-        {% else -%}
+        {% elif code not in ['ast', 'cak', 'dsb', 'hsb', 'kab', 'lij', 'son'] -%}{#- Bug 1364470: drop locales not supported by Google -#}
```

Comment:
Looks like the list isn't quite the same. These are the 3 letter locales in `settings.PROD_LANGUAGES`:

```python
['ach', 'ast', 'cak', 'dsb', 'hsb', 'kab', 'lij', 'ltg', 'mai', 'son']
```

But perhaps ach, ltg, and mai just aren't on any of these pages?